### DISCUSSION TITLE
End of life for Identity Server 4

### DISCUSSION THREAD

**Discussion by kingjordan** [jasontaylordev/CleanArchitecture#764] [General]
Seeing as identity server 4 has reached end of life, is there plans to migrate to the duendesoftware edition of identity server?

**Comment by hummer007** [comment]
another free library is [OpenIddict ](https://github.com/openiddict/openiddict-core)

**Comment by quintonv** [comment]
I think OpenIddict would be a better alternative to duendesoftware as that requires a very expensive license for production usage.
